packages/console/src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactNode, RefObject, useRef } from 'react';
+import ReactModal from 'react-modal';
+
+import * as styles from './index.module.scss';
+import usePosition from './use-position';
+
+export { default as DropdownItem } from './DropdownItem';
+
+type Props = {
+  children: ReactNode;
+  title?: ReactNode;
+  isOpen: boolean;
+  onClose?: () => void;
+  anchorRef: RefObject<HTMLElement>;
+  isFullWidth?: boolean;
+};
+
+const Dropdown = ({ children, title, isOpen, onClose, anchorRef, isFullWidth }: Props) => {
+  const overlayRef = useRef<HTMLDivElement>(null);
+
+  const { position, mutate } = usePosition(anchorRef, overlayRef);
+
+  return (
+    <ReactModal
+      shouldCloseOnOverlayClick
+      isOpen={isOpen}
+      style={{
+        content: position
+          ? {
+              left: `${position.left}px`,
+              top: `${position.top + 4}px`,
+              width: isFullWidth ? position.width : undefined,
+            }
+          : { visibility: 'hidden' },
+      }}
+      className={styles.content}
+      overlayClassName={styles.overlay}
+      onRequestClose={onClose}
+      onAfterOpen={() => {
+        mutate();
+      }}
```

Comment:
nit: `onAfterOpen={mutate}` would be a little bit better

Reply:
updated